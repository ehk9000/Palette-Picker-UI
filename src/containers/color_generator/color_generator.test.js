import React from 'react';
import { shallow } from 'enzyme';
import { ColorGenerator, mapDispatchToProps, mapStateToProps } from './index';

describe('Color Generator', () => {
  let wrapper;
  let mockProjects = { id: 1, name: 'Test' }
  let mockPalettes = { id: 1, color_1: "SDFH", name: 'wave' }
  let mockCurrentPalette = { id: 1, color_1: "SDFH", name: "wave" }

  beforeEach(() => {
    wrapper = shallow ( <ColorGenerator
      projects={mockProjects}
      palettes={mockPalettes}
      currentPalette={mockCurrentPalette}
    /> )
  });

  it('Should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
