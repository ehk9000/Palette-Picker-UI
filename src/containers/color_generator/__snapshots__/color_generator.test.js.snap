// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Generator Should render correctly 1`] = `ShallowWrapper {}`;
